compose-table/src/main/java/org/dhis2/composetable/ui/DataSetTableUi.kt

```diff
@@ -204,7 +210,9 @@ fun TableItemRow(
                 onClick = onClick
             )
         }
-        Divider(modifier = Modifier.fillMaxWidth())
+        if (isNotLastRow) {
```

Comment:
Here also, maybe the model `TableRowModel` can has a property called fillMaxWidth: Boolean. And tested in the mapper.